rosys/hardware/gnss.py

```diff
@@ -318,6 +320,7 @@ def simulate(self) -> None:
 
         self.last_measurement = GnssMeasurement(
             time=rosys.time(),
+            gnss_time=rosys.time(),
```

Comment:
```suggestion
        timestamp = rosys.time()
        self.last_measurement = GnssMeasurement(
            time=timestamp,
            gnss_time=timestamp,
```

How about this to make sure the timestamps are the same